Source/JavaScriptCore/inspector/remote/RemoteControllableTarget.h

```diff
@@ -41,7 +42,7 @@ class FrontendChannel;
 
 using TargetID = unsigned;
 
-class RemoteControllableTarget {
+class JS_EXPORT_PRIVATE RemoteControllableTarget : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<RemoteControllableTarget> {
 public:
     JS_EXPORT_PRIVATE RemoteControllableTarget();
```

Comment:
We need to remove JS_EXPORT_PRIVATE here and on the 3 lines below, for the Windows build.